Tools-Override/tests.targets

```diff
@@ -28,13 +28,15 @@
 
   <PropertyGroup>
     <XunitRuntimeConfig>$(ToolsDir)\xunit.console.netcore.runtimeconfig.json</XunitRuntimeConfig>
-    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT' and ('$(BuildingInsideVisualStudio)' != 'true' or '$(RUNTIME_PATH)' != '')">%RUNTIME_PATH%\</TestRuntimeEnvVar>
     <TestRuntimeEnvVar Condition="'$(OS)' != 'Windows_NT'">$RUNTIME_PATH/</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(TestRuntimeEnvVar)' == ''">$(TestSharedFxDir)\</TestRuntimeEnvVar>
```

Comment:
@weshaggard, @mellinoe Do you understand what I mean? Someone will inject some switch somewhere in the middle here and the start program will be busted- but not until you fully clean.

That brings up another thing- shouldn't our overrides be pulled in automatically (by include) rather than copied? Or if not, shouldn't we be copying over newer files?